README.md

```diff
@@ -0,0 +1,91 @@
+# HTML Form Validator
+
+This project is based on [a tweet from @Ocramius](https://twitter.com/Ocramius/status/680817040429592576), I hope this is what he was thinking.
+
+## TODO
+- Decide how rules should be defined on a input-field.
+    - How should we handle validator arguments i.e. 'greaterthan'
+    - How should we handle multiple validators on a single form input
+- Decide whether we should keep returning the Zend\InputFilter\InputFilter instance or create a simplified wrapper, upon validation
+- Implement method for adding custom validators
+- Add issues with API suggestions
+
+## Basic usage
+
+```php
+$formHTML = '
+    <div class="form-container">
+        <form action="/make-this-happen" method="post">
+            <label for="email">Email:</label>
+            <input
+                type="email"
+                id="email"
+                name="email"
+                value=""
+                data-validator="emailaddress"
+                required="required"
+            />
+            <input type="submit"/>
+        </form>
+    </div>
+';
+
+
+$formValidator = new Validator($formHTML);
+
+// simply an instance of [Zend\InputFilter\InputFilter](https://github.com/zendframework/zend-inputfilter/blob/master/src/InputFilter.php)
+$validationResponse = $formValidator->validate($_POST);
+
+// determine validity
+if($validationResponse->isValid()) {
+	echo "Succeed in life.";
+} else {
+    // die and dump the messages
+    die(var_dump($validationResponse->getMessages()));
+}
+
+```
+
+## "Advanced" usage
+
+```php
+$formHTML = '
+    <div class="form-container">
+        <form action="/make-totally-different-form" method="get">
+            <input type="hidden" name="foo" value="bar">
+        </form>
+        <form action="/make-this-happen" method="post" id="important">
+            <label for="email">Email:</label>
+            <input
+                type="email"
+                id="email"
+                name="email"
+                value=""
+                data-validator="emailaddress"
+                required="required"
+            />
+            <input type="submit"/>
+        </form>
+    </div>
+';
+
+
+// specify which form you want
+$formValidator = new Validator($formHTML, '#important');
```

Comment:
@Ocramius the selector can be omitted and if it is, it will find the first form in the document.
In the upcoming commits I'll add proper exceptions, if no form is found.